src/test/java/net/minestom/server/inventory/InventoryTest.java

```diff
@@ -78,9 +78,9 @@ public void testAdd() {
 
     @Test
     public void testIds() {
-        for (int i = 0; i <= 256; ++i) {
-            final Inventory inventory = new Inventory(InventoryType.CHEST_1_ROW, "title");
-            assertTrue(inventory.getWindowId() != 0);
+        for (int i = 0; i <= 1000; ++i) {
+            final int windowId = new Inventory(InventoryType.CHEST_1_ROW, "title").getWindowId();
```

Comment:
why are you using an `int` to store windowId, when getWindowId returns a byte?

Reply:
I tried to not touch base variables. The previous contributor used atomic integer. I do not believe there is any atomic byte. Please correct me if I am wrong.

Do you have any better idea to fix this?

Reply:
`final byte windowId = new Inventory(InventoryType.CHEST_1_ROW, "title").getWindowId();`

Reply:
> `final byte windowId = new Inventory(InventoryType.CHEST_1_ROW, "title").getWindowId();`

Oh, you were talking about the test I guess. Window/inventory packets request window id as a byte. That's why I use byte everywhere for window ids.

Reply:
it's an integer here though

Reply:
> in the line you wrote, it's an integer

Oh, I am so sorry, I didn't see it 😢